esmvaltool/recipes/recipe_portrait_CMIP.yml

```
@@ -0,0 +1,209 @@
+# ESMValTool
+#
+---
+documentation:
+  title: Performance metrics plots.
+  description: >
+    Compare performance of CMIP simulations to a reference dataset.
+  authors:
+    - cammarano_diego
+    - lindenlaub_lukas
+  maintainer:
+    - lindenlaub_lukas
+  references:
+    - eyring21ipcc
+    - gleckler08jgr
+
+cmip5: &CMIP5
+  project: CMIP5
+  ensemble: r1i1p1
+
+datasets:
+  # cmip5
+  - {<<: *CMIP5, dataset: ACCESS1-0}
+  - {<<: *CMIP5, dataset: CESM1-BGC}
+  - {<<: *CMIP5, dataset: CNRM-CM5}
+  - {<<: *CMIP5, dataset: GFDL-ESM2M}
+  - {<<: *CMIP5, dataset: HadGEM2-CC}
+  - {<<: *CMIP5, dataset: IPSL-CM5B-LR}
+  - {<<: *CMIP5, dataset: MIROC-ESM}
+  - {<<: *CMIP5, dataset: MPI-ESM-LR}
+  - {<<: *CMIP5, dataset: MRI-CGCM3}
+  # cmip6
+  - {dataset: ACCESS-ESM1-5, institute: CSIRO}
+  - {dataset: CESM2, institute: NCAR}
+  - {dataset: CNRM-CM6-1, ensemble: r1i1p1f2, grid: gr}
+  - {dataset: GFDL-CM4, grid: gr1}
+  - {dataset: MIROC-ES2L, ensemble: r1i1p1f2}
+  - {dataset: MPI-ESM1-2-LR}
+  - {dataset: MRI-ESM2-0}
+  - {dataset: UKESM1-0-LL, ensemble: r1i1p1f2}
+
+preprocessors:
+  default: &default  # common preprocessor settings
+    regrid:
+      target_grid: 3x3
+      scheme: linear
+    distance_metric:
+      metric: weighted_rmse
+    climate_statistics:
+      operator: mean
+      period: month
+    mask_fillvalues:
+      threshold_fraction: 0.95
+    multi_model_statistics:
+      span: overlap
+      statistics: [mean, median]
```

```suggestion
      statistics:
        - operator: mean
        - operator: percentile
          percent: 50
```
It may be nice to compute the 50th percentile instead of the median, because median is not yet lazy. See https://github.com/SciTools/iris/pull/6167.